homeassistant/components/zwave_me/number.py

```diff
@@ -42,6 +42,8 @@ class ZWaveMeNumber(ZWaveMeEntity, NumberEntity):
     @property
     def value(self):
         """Return the unit of measurement."""
+        if self.device.level == 99:  # Scale max value
```

Comment:
Maybe set `_attr_max_value` to 99 instead?

Reply:
I don't know which approach is better in HA. Please suggest.

Reply:
We don't have a certain standard here. It depends on what the number entity represents.

Reply:
If this is also okay, let's keep as is